src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -1147,6 +1147,58 @@ def test_containerapp_scale_update(self, resource_group):
             JMESPathCheck("properties.template.scale.rules[0].custom.auth[1].secretRef", "app-key"),
         ])
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="westeurope")
+    def test_containerapp_scale_update_azure_queue(self, resource_group):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+
+        app = self.create_random_name(prefix='aca', length=24)
+        env = prepare_containerapp_env_for_app_e2e_tests(self)
+
+        self.cmd(f'containerapp create -g {resource_group} -n {app} --image nginx --ingress external --target-port 80 --environment {env} --scale-rule-name http-scale-rule --scale-rule-http-concurrency 50 --scale-rule-auth trigger=secretref --scale-rule-metadata key=value', checks=[JMESPathCheck("properties.template.scale.rules[0].http.metadata.key", "")])
+
+        self.cmd(f'containerapp show -g {resource_group} -n {app}', checks=[
+            JMESPathCheck("properties.template.scale.rules[0].name", "http-scale-rule"),
+            JMESPathCheck("properties.template.scale.rules[0].http.metadata.concurrentRequests", "50"),
+            JMESPathCheck("properties.template.scale.rules[0].http.metadata.key", "value"),
+            JMESPathCheck("properties.template.scale.rules[0].http.auth[0].triggerParameter", "trigger"),
+            JMESPathCheck("properties.template.scale.rules[0].http.auth[0].secretRef", "secretref"),
+            JMESPathCheck("properties.template.scale.rules[0].http.metadata.key", "value"),
+        ])
+        queue_name = self.create_random_name(prefix='queue', length=24)
+        containerapp_yaml_text = f"""
+properties:
+    template:
+        containers:
+        -   image: nginx
+            name: azure-equeue-container
+            resources:
+              cpu: 0.5
+              memory: 1Gi
+              ephemeralStorage: 2Gi
+        scale:
+            minReplicas: 0
+            maxReplicas: 1
+            rules:
+            - name: azure-queue-scale-rule
+              azureQueue:
+                queueName: {queue_name}
+                queueLength: 1
+                auth:
+                - secretRef: azure-storage
+                  triggerParameter: connection
+"""
+        containerapp_file_name = f"{self._testMethodName}_containerapp.yml"
+
+        write_test_file(containerapp_file_name, containerapp_yaml_text)
+        self.cmd(f'containerapp update -n {app} -g {resource_group} --yaml {containerapp_file_name}', checks=[
```

Comment:
Since it seems the change also applies to the update command, It is suggested to add a check for properties.template.scale.rules[0].http.metadata.key here as well.